mesonbuild/compilers/c.py

```diff
@@ -838,16 +834,29 @@ def thread_link_flags(self, env):
             return []
         return ['-pthread']
 
+    def linker_to_compiler_args(self, args):
+        return args
+
+    def has_arguments(self, args, env, code, mode):
+        return self.compiles(code, env, extra_args=args, mode=mode)
+
     def has_multi_arguments(self, args, env):
         for arg in args:
             if arg.startswith('-Wl,'):
-                mlog.warning('''{} looks like a linker argument, but has_argument
-and other similar methods only support checking compiler arguments.
-Using them to check linker arguments are never supported, and results
-are likely to be wrong regardless of the compiler you are using.
-'''.format(arg))
-        return self.compiles('int i;\n', env, extra_args=args)
+                mlog.warning('{} looks like a linker argument, '
+                    'but has_argument and other similar methods only support '
+                    'checking compiler arguments. Using them to check linker '
+                    'arguments are never supported, and results are likely '
```

Comment:
__[Flake8]__

[E128] continuation line under-indented for visual indent


<!-- Comment by [SideCI](https://sideci.com) -->